code/Modules/Input/emsc/emscInputMgr.cc

```diff
@@ -82,6 +85,124 @@ emscInputMgr::discardCallbacks() {
     emscripten_set_touchcancel_callback("#canvas", 0, true, 0);
     emscripten_set_devicemotion_callback(0, true, 0);
     emscripten_set_deviceorientation_callback(0, true, 0);
+    emscripten_set_gamepaddisconnected_callback(this, true, emscGamepadDisconnected);
+}
+
+//------------------------------------------------------------------------------
+void
+emscInputMgr::updateGamepadsAttached(int& gamepadsAttached) {
```

Comment:
Got in the habit of using return values strictly for error codes a while back and used references for handing data out when possible while allowing ownership of the thing being filled to reside outside. 
Have updated it to use a return value.